NorthstarDLL/color.cpp

```diff
@@ -0,0 +1,26 @@
+#include "pch.h"
+
+// clang-format off
+namespace NS::Colors
+{
+	Color SCRIPT_UI		(100, 255, 255);
+	Color SCRIPT_CL		(100, 255, 100);
+	Color SCRIPT_SV		(255, 100, 255);
+	Color NATIVE_UI		(50 , 150, 150);
+	Color NATIVE_CL		(50 , 150, 50 );
+	Color NATIVE_SV		(150, 50 , 150);
+	Color NATIVE_EN		(252, 133, 153);
+	Color FILESYSTEM	(0  , 150, 150);
+	Color RPAK			(255, 190, 0  );
+	Color NORTHSTAR		(66 , 72 , 128);
+	Color ECHO			(150, 150, 159);
+
+	Color TRACE		(0  , 255, 255);
+	Color DEBUG		(0  , 255, 255);
+	Color INFO		(16 , 160, 16 );
```

Comment:
I personally would prefer info to be a more subtle colour as it doesn't signify anything of importance, I don't mind the green though